### PR TITLE
POM | AT_13.02_006 | Pipeline | Delete created project - Verify the user can cancel the deletion of the project in the confirmation window.

### PR DESCRIPTION
**Pull request by NinaPrud**
https://trello.com/c/siEpZGh2/1831-pom-at1302006-pipeline-delete-created-project-verify-the-user-can-cancel-the-deletion-of-the-project-in-the-confirmation-window

### CODE REVIEW

**Review by KseniyaParfianovich — COMMENTED** [review]
Don't forget to skip the same tests, not rewritten in POM

**Review by KseniyaParfianovich — APPROVED** [review]

**Comment by NinaPrud** [comment]
> Don't forget to skip the same tests, not rewritten in POM

This is a new test, not a refactoring

File: cypress/e2e/tests/pipelineProject.cy.js [hunk 1]
```diff
@@ -112,4 +112,16 @@ describe('pipelineProject',()=>{
...
+    it('AT_13.02_006 | Pipeline - Verify the user can cancel the deletion of the pipeline in the confirmation window.',()=>{
+        cy.createPipeline(newItemPageData.pipelineName); 
+
+        homePage
+          .clickPipelineProjectName(newItemPageData.pipelineName)
+          .clickDeletePipelineBtn()
+          .clickCancelConfirmDeletePipeline()
+          .verifyPipelinePageUrl()
+          .getPipelinePageHeadline()
+          .should('have.text', `${pipelinePageData.pipelinePageHeaderStart} ${newItemPageData.pipelineName}`)
```
[the remark this answers is not in the document]
Comment (NinaPrud) [reply cypress/e2e/tests/pipelineProject.cy.js]: fixed

[the remark this answers is not in the document]
Comment (NinaPrud) [reply cypress/e2e/tests/pipelineProject.cy.js]: fixed

**Review by Nadzeya-Yuchko — APPROVED** [review]

**Pull request merged**